### PR TITLE
Remove repeated page objects methods #5531

### LINKED ISSUE #5531: Remove repeated page objects methods
Many methods in the package `test.pageobjects` are repetitions of each other. Some of them:
- `createCorrectLoginPageType`
- `createNewPage`
- Defining own `WebDriverWait` and `wait.until` when abstractions exist via `AppPage`
- Defining own navbar tab links, similarly

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #5531 

The login for admin UI tests was created before (by yours truly) to ensure the correct admin username for the test pages. Turns out it's easily achieved by masquerading.

Everything else is either duplicate or unnecessary.

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 1]
```diff
@@ -186,10 +176,18 @@ public AppPage navigateTo(Url url) {
      * the type indicated by the parameter {@code newPageType}.
      */
     public <T extends AppPage> T changePageType(Class<T> newPageType) {
-        return createNewPage(browser, newPageType);
+        return getNewPageInstance(browser, newPageType);
     }
 
     /**
+     * Gives a LoginPage instance based on the given Browser and test configuration.
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: @JosephineKwa I'll add "Fails if the new page is not a valid login page" here later.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java [hunk 2]
```diff
@@ -411,16 +408,9 @@ public String getResponseValue(String courseId, String sessionName) {
...
-            wait.until(ExpectedConditions.textToBePresentInElement(
-                    browser.driver.findElement(
-                            By.xpath("//tbody/tr[" + (int) (sessionRowId + 1)
-                            + "]/td[contains(@class,'session-response-for-test')]")),
-                            responseRate));
-        } catch (TimeoutException e) {
-            fail("Not expected message");
-        }
+        waitForTextContainedInElementPresence(
+                By.xpath("//tbody/tr[" + (sessionRowId + 1) + "]/td[contains(@class,'session-response-for-test')]"),
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java]: For clarity, perhaps you could css instead of xpath to find the element

**Comment by JosephineKwa** [comment]
Aside from the comment above, it looks good

File: src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java (hunk 2 above)
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java]: screen shot 2016-06-06 at 11 52 23 am

I agree that css is clearer, but I'm causing inconsistency if I do that here. We can fix this any other time.

**Pull request merged**